Sources/Commands/SwiftTestTool.swift

```diff
@@ -870,6 +933,7 @@ fileprivate func constructTestEnvironment(
         // the filter.
         let codecovProfile = buildParameters.buildPath.appending(components: "codecov", "default%m.profraw")
         env["LLVM_PROFILE_FILE"] = codecovProfile.asString
+        clearDirectory(path: buildParameters.buildPath.appending(components: "codecov"))
```

Comment:
We should do this wherever we run the test action, not as a side effect of this method.